### PR TITLE
new KB - Eliminate the Last Item's stretching in DataLayout

### PR DESCRIPTION
**Pull request by dessyordanova**

### CODE REVIEW

File: knowledge-base/stretched-last-item-in-datalayout.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            InitializeComponent();
+            this.radDataLayout1.DataSource = new Employee()
+            {
+                FirstName = "Sarah",
+                LastName = "Blake",
+                Occupation = "Supplied Manager",
+                StartingDate = new DateTime(2005, 04, 12),
+                IsMarried = true
+            };
+            int height = this.radDataLayout1.LayoutControl.Controls.Count * 20;
```
Comment (TodorVyagov) [inline knowledge-base/stretched-last-item-in-datalayout.md:43]: There are two controls added by default in RadLayoutControl - the horizontal and vertical scrollbars, so to have precise calculations, we need to subtract them.
In addition, the default size for editors in the Fluent theme is 24 pixels, so this increases the height per layout control item.
int height = (this.radDataLayout1.LayoutControl.Controls.Count - 2 ) * 24;

**Review by TodorVyagov — CHANGES_REQUESTED** [review]
There are two controls added by default in RadLayoutControl - the horizontal and vertical scrollbars, so to have precise calculations, we need to subtract them.
In addition, the default size for editors in the Fluent theme is 24 pixels, so this increases the height per layout control item.
int height = (this.radDataLayout1.LayoutControl.Controls.Count - 2 ) * 24;

File: knowledge-base/stretched-last-item-in-datalayout.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+position: 0
+tags: datalayout, last, vertical, stretch, space
+res_type: kb
+---
+
+## Environment
+ 
+|Product Version|Product|Author|
+|----|----|----|
+|2023.2.606|RadDataLayout for WinForms|[Dinko Krastev](https://www.telerik.com/blogs/author/desislava-yordanova)|
```
Comment (TodorVyagov) [inline knowledge-base/stretched-last-item-in-datalayout.md:16]: [Dinko Krastev]

**Comment by dessyordanova** [comment]
Addressed the feedback.

**Review by TodorVyagov — APPROVED** [review]

**Pull request merged**